test/common/fit_common.py

```diff
@@ -867,7 +869,7 @@ def node_select():
             if str(fitargs()['sku']) in json.dumps(skuentry):
                 skuid = skuentry['id']
         # Collect node IDs
-        catalog = rackhdapi('/api/2.0/nodes')
+        catalog = rackhdapi('/api/2.0/nodes?type=compute')
```

Comment:
We need to do a sort on the returned nodes for using the 'index' command.
Different calls of run_tests.py that use the passed in index needs to use the node list in a specific order, otherwise, index is random.
ie. sort the return of api/2.0/nodes into 'catalog'